.github/workflows/test-cli.yml

```diff
@@ -0,0 +1,32 @@
+name: CLI Test
```

Comment:
As this workflow is named `CLI test` with file named `test-cli.yml` should we also rename the existing `test.yml` file to `test-template.yml` and rename the existing test workflow to `Test Template`?

Reply:
Tentative done in 0f23510 :) 
I've matched the exact name of the packages sub-folders (`cli-tool` and `cra-template`).